jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java

```diff
@@ -335,6 +334,7 @@ private void closeCallbacks() {
         cursorPosCallback.close();
         scrollCallback.close();
         mouseButtonCallback.close();
+        context.removeWindowSizeListener(windowSizeListener);
```

Comment:
I wonder what will happen if the user adds some window listeners explicitly, should we remove them too ??

Reply:
Oh okay, looking around again I get this, this removes the callback for the glfwMouseInput only.

Reply:
Yes.